src/wallet/wallet.cpp

```diff
@@ -2503,7 +2503,7 @@ CAmount CWallet::GetUnconfirmedBalance() const
         for (std::map<uint256, CWalletTx>::const_iterator it = mapWallet.begin(); it != mapWallet.end(); ++it)
         {
             const CWalletTx* pcoin = &(*it).second;
-            if (!pcoin->IsTrusted() && pcoin->GetDepthInMainChain() == 0 && !pcoin->IsLockedIX() && pcoin->InMempool())
+            if (!pcoin->IsTrusted())
```

Comment:
Err... That's not what I meant, I meant that addition of `!pcoin->IsLockedIX()` was excessive because it's already a part of `IsTrusted()`, not that smth else should be removed..

Reply:
Actually... Maybe I double checked and I guess I was wrong even in my initial comment - we don't really check if ix is final or not when MN votes for some input so `IsTrusted() == false && IsLockedIX() == true` could be a thing probably. Pls revert this (and the one below) to your original version. Sorry 😇